Formula/deno.rb

```diff
@@ -98,6 +105,15 @@ def install
       system "ninja", "-C", "out"
     end
 
+    resource("tinycc").stage buildpath/"tinycc"
+    cd "tinycc" do
+      ENV.append_to_cflags "-fPIE" if OS.linux?
```

Comment:
AFAIK this means it's doing a static build instead of shared

Reply:
Yes, vendors its own `libtinycc`. Not cool.